lib/go-tc/types.go

```diff
@@ -49,6 +50,31 @@ type TypeNullable struct {
 	UseInTable  *string    `json:"useInTable" db:"use_in_table"`
 }
 
+// TypesResponseV5 is the type of a response from Traffic Ops to a GET request
```

Comment:
Can you added why you created V5 struct? like `TypesResponseV5 is the type of a response (for RFC3339) from Traffic Ops to a GET Request`